lib/mayaUsd/render/MaterialXGenOgsXml/libraries/mx39_lighting_maya_all.glsl

```diff
@@ -0,0 +1,112 @@
+#ifdef MAYA_MX39_USING_ENVIRONMENT_FIS
```

Comment:
Same code as found in the regular Maya lighting files, but adapted to use Mx39FresnelData and mx39compute_fresnel.